app/models/user.rb

```diff
@@ -23,7 +23,36 @@ def member_of?(organization_id)
   end
 
   def domain_name
-    email[/@(?<domain>[^\.]*).(.*)/,"domain"]
+    email[/@(?<domain>[^\.]*).(.*)/,"domain"] if email
+  end
+
+  def organization_names
+    names = ""
+    organizations.find_each { |o| names += "#{o.name}," }
```

Comment:
I'd probably just do a ``organizations.map`` instead, unless there could be hundreds of organizations.